src/client/Microsoft.Identity.Client/Internal/Broker/BrokerSilentRequest.cs

```diff
@@ -68,7 +69,9 @@ internal void CreateRequestParametersForBroker()
             string extraQP = string.Join("&", _authenticationRequestParameters.ExtraQueryParameters.Select(x => x.Key + "=" + x.Value));
             BrokerPayload.Add(BrokerParameter.ExtraQp, extraQP);
             BrokerPayload.Add(BrokerParameter.ExtraOidcScopes, BrokerParameter.OidcScopesValue);
-            BrokerPayload.Add(BrokerParameter.LoginHint, _silentParameters.LoginHint);
+            BrokerPayload.Add(BrokerParameter.HomeAccountId, _silentParameters.Account?.HomeAccountId?.Identifier);
+            BrokerPayload.Add(BrokerParameter.LocalAccountId, _silentParameters.Account?.HomeAccountId?.ObjectId);
+            BrokerPayload.Add(BrokerParameter.Username, !string.IsNullOrEmpty(_silentParameters.Account?.Username) ? _silentParameters.Account?.Username : _silentParameters.LoginHint);
```

Comment:
Login hint and username are interchangeable here. Also updating the broker parameter to username.